### PR TITLE
feat(content): add description for Hannes Heine

### PR DESCRIPTION
**Pull request by Elweyn**
## 🍰 Pullrequest

Add a page for the person Hannes Heine.

### Issues

- None

### Todo

- Curriculum Vitae
- [X] Add a photo
- [x] Add a description of the person
- [x] Add project in which he worked
- [x] Add development skils
- [x] Add framework skills

### CODE REVIEW

File: docs/people/hannes-heine.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# Hannes Heine
+
+## Presentation
+
+![Hannes Heine](~@images/portrait/hannes-heine.jpg)
+
+I was born in germany and raised in france, so I speak both languages fluently.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:7]:
```suggestion
I was born in Germany and raised in France, so I speak both languages fluently.
```

File: docs/people/hannes-heine.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+As child except for my interest for sports I always excelled in sciences. In 10th grade I came across coding, the starting time was so exiting I had the impression that I could do magic on a small device. It captivated me so much that after the graduation in 2008 (baccalauréat) I studied for 3 years in the beautiful city of Toulouse, in the fields of mathematics, mechanics and informatics.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:9]:
```suggestion
As child except for my interest for sports I always excelled in sciences. In 10th grade I came across coding, the starting time was so exiting I had the impression that I could do magic on a small device. It captivated me so much, that after the graduation in 2008 (baccalauréat), I studied for 3 years in the beautiful city of Toulouse, in the fields of mathematics, mechanics and informatics.
```

File: docs/people/hannes-heine.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+After that I moved back to germany in the city were I was born Hamburg. Their I worked for two different programming companies (White Paper Media ltd. and MSP AG) until 2017 were I made myself a freelancer.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:11]:
```suggestion
After that I moved back to Germany in the city were I was born Hamburg. Their I worked for two different programming companies (White Paper Media ltd. and MSP AG) until 2017 when I made myself a freelancer.
```

File: docs/people/hannes-heine.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+As freelancer I worked on different projects: For the DeutscheSee were I supported them for the shop development, Human Connection were I helped to translate from german to french, Gradido were I do tasks from backend to frontend, and I setup small Shopware shops.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:13]:
```suggestion
As freelancer I worked on different projects: For the DeutscheSee were I supported them for the shop development, Human Connection were I helped to translate from German to French, Gradido were I do tasks from backend to frontend, and I setup small Shopware shops.
```

File: docs/people/hannes-heine.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+In my free time I like to play video games, do some reiki and all kind of sports. I travel in south france and germany.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:15]:
```suggestion
In my free time I like to play video games, do some reiki and all kind of sports. I travel in southern France and Germany.
```

File: docs/people/hannes-heine.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+
+You can find me on [Github](https://github.com/elweyn)
+
+## Role
+
+- Backend Developer
+- DevOps
+- Softwarearchitekt
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:23]:
```suggestion
- Softwarearchitect
```

File: docs/people/hannes-heine.md [hunk 7]
```diff
@@ -0,0 +1,83 @@
...
+- [Express](https://expressjs.com/)
+- [Apollo](https://www.apollographql.com/docs/apollo-server/)
+- [Angular](https://angular.io/)
+
+### For Java
+
+- [JAX-RS](https://www.oracle.com/technical-resources/articles/java/jax-rs.html)
+- [Apache POI](https://poi.apache.org/)
+- JavaPOS
+- JDBC
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:51]: links?

File: docs/people/hannes-heine.md [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+
+- [shopware](https://www.shopware.com/)
+- [Wordpress](https://wordpress.org/)
+
+### Web basics
+
+- [HTML](https://en.wikipedia.org/wiki/HTML)
+- [CSS](https://en.wikipedia.org/wiki/CSS)
+- [Bootstrap](https://getbootstrap.com/)
+- XSLT
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:63]: link?

File: docs/people/hannes-heine.md (hunk 4 above)
Comment (Elweyn) [reply docs/people/hannes-heine.md]: "german to french" why write that in capitalcase?

File: docs/people/hannes-heine.md [hunk 9]
```diff
@@ -0,0 +1,111 @@
+# Hannes Heine
+
+## Presentation
+
+![Hannes Heine](~@images/portrait/hannes-heine.jpg)
+
+I was born in Germany and raised in France, so I speak both languages fluently. In my free time I like to play video games, do Reiki and all kind of sports.
+As child except for my interest for sports I always excelled in sciences. In 10th grade I came across coding, the starting time was so exiting I had the impression that I could do magic on a small device. It captivated me so much that after my graduation in 2008 (baccalauréat), I studied for 3 years in the beautiful city of Toulouse (mathematics, mechanics and informatics).
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:8]:
```suggestion
As child, except for my interest for sports, I always excelled in sciences. In 10th grade I came across coding, the starting time was so exiting, I had the impression that I could do magic on a small device. It captivated me so much that after my graduation in 2008 (baccalauréat), I studied for 3 years in the beautiful city of Toulouse (mathematics, mechanics and informatics).
```

File: docs/people/hannes-heine.md [hunk 10]
```diff
@@ -0,0 +1,111 @@
...
+After that I moved back to Germany in the city were I was born Hamburg.
```
Comment (ulfgebhardt) [inline docs/people/hannes-heine.md:9]:
```suggestion
After that I moved back to Germany in the city were I was born - Hamburg.
```

**Review by ulfgebhardt — APPROVED** [review]

**Pull request merged**